src/include/simeng/models/outoforder/Core.hh

```diff
@@ -44,12 +47,16 @@ class Core : public simeng::Core {
   const ArchitecturalRegisterFileSet& getArchitecturalRegisterFileSet()
       const override;
 
+  /** Send a syscall to the system syscall handler. */
+  void sendSyscall(OS::SyscallInfo syscallInfo) const override;
```

Comment:
It still says system while the one in simeng/Core.hh says simulated Operating System's

Reply:
Outdated